### ISSUE TITLE
the MOTA is very low

### ISSUE THREAD

**Issue by wxy932** [gamzekiprit/GCNNMatch.v1#2]
Thanks for your contribution!!
I trained the GCNNMatch model on MOT17 dataset, the detector adopted FasterR-CNN, but the tracking performance is poor.
The MOTA only is 40.3.
So I want to ask what dataset you used for training, several datasets or only MOT17  dataset? What's the MOTA value?
I need to how to adjust parameters so that I can get the better tracking performance?

**Comment by gamzekiprit** [comment]
Hello, 
yes, I also trained the model on MOT17 dataset on MOT20 it doesn't work anyway. However, since uploading the model to the MOTChallenge server is very time consuming to get MOTA and also because of memory issues, I first divided the training set of MOT17 into training , validation and a test set and used the py-motmetrics library for the evaluation. I also dont have the results they mention in the paper, but from my side it was mainly because of memory issues I could not use whole MOT17 training set. 

If I were you, I would not try to adjust the hyperparameters first, as this would only have an effect of maybe a bit. Which feature extractor do you use?  If you use Densenet121, this could be the problem, as it did not converge for me either (i dont really know why).
